webpack.js

```diff
@@ -0,0 +1,18 @@
+const { merge } = require('webpack-merge')
+const path = require('path')
+const webpackConfig = require('@nextcloud/webpack-vue-config')
+
+const config = {
+	entry: {
+		admin: path.join(__dirname, 'src', 'admin.js'),
+		user: path.join(__dirname, 'src', 'user.js'),
+		public: path.join(__dirname, 'src', 'public.js'),
+	},
+	output: {
+		filename: 'terms_of_service_[name].js',
```

Comment:
Why?

Reply:
Using the default should be fine, no?

Reply:
So I dont need to change php code :see_no_evil: and the default would be `terms_of_service-[name].js`